Client/Assets/Interstitial Pages/Pages/GenericError.html

```diff
@@ -18,9 +18,9 @@
             if (fresh) {
                 fresh = false;
             } else {
-                webkit.messageHandlers.localRequestHelper.postMessage({
-                    securitytoken: "%security_token%",
-                    type: "reload"
+                webkit.messageHandlers.%message_handler%.postMessage({
```

Comment:
just a suggestion: you can reference these using ["%message_handler%"] while not that different, it just doesn't give you warnings when editing the code. But unsure if there are any other consequences or referencing in that way.

Reply:
No difference, but fixed :)